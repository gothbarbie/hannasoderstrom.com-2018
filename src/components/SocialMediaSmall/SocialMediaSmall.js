import React from 'react'
import styled from 'styled-components'

const Section = styled.section`
  display: flex;
  justify-content: center;
  align-items: center;

  & ul {
    display: flex;
    justify-content: center;
    align-items: center;
  }

  & li {
    margin: 0 4vw 0 0;
  }

  & a {
    color: #fff;
    display: flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
  }
`
const Icon = styled.i`
  font-size: 2rem;
  color: #fff;
`

const SocialMediaSmall = () => (
  <Section>
    <ul>
      <li>
        <a href="https://github.com/gothbarbie/hannasoderstrom.com-2018">
          <Icon className="fab fa-github" />
        </a>
      </li>
      <li>
        <a href="https://www.linkedin.com/in/hannasoderstromwebdev/">
          <Icon className="fab fa-linkedin" />
        </a>
      </li>
      <li>
        <a href="https://medium.com/@gothbarbie84">
          <Icon className="fab fa-medium" />
        </a>
      </li>
    </ul>
  </Section>
)

export default SocialMediaSmall
