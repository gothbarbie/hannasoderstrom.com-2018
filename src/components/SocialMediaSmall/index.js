import SocialMediaSmall from './SocialMediaSmall'
export default SocialMediaSmall
