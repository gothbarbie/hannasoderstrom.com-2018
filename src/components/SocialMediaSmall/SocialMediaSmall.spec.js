import React from 'react'
import { shallow } from 'enzyme'

import SocialMediaSmall from './index'

describe('components/SocialMediaSmall', () => {
  let component

  beforeEach(() => {
    component = shallow(<SocialMediaSmall />)
  })

  it('renders', () => {
    expect(component).toMatchSnapshot()
  })
})
