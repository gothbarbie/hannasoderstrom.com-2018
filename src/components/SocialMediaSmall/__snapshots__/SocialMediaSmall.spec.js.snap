// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SocialMediaSmall renders 1`] = `
<styled.section>
  <ul>
    <li>
      <a
        href="https://github.com/gothbarbie/hannasoderstrom.com-2018"
      >
        <styled.i
          className="fab fa-github"
        />
      </a>
    </li>
    <li>
      <a
        href="https://www.linkedin.com/in/hannasoderstromwebdev/"
      >
        <styled.i
          className="fab fa-linkedin"
        />
      </a>
    </li>
    <li>
      <a
        href="https://medium.com/@gothbarbie84"
      >
        <styled.i
          className="fab fa-medium"
        />
      </a>
    </li>
  </ul>
</styled.section>
`;
